includes/controllers/class.llms.controller.lesson.progression.php

```diff
@@ -127,7 +127,16 @@ public function handle_complete_form() {
 			return;
 		}
 
-		do_action( 'llms_trigger_lesson_completion', get_current_user_id(), $lesson_id, 'lesson_' . $lesson_id );
+		/**
+		 * Filter to modify the User id instead of current loggedin user id.
+		 *
+		 * @param int $user_id User id to mark lesson as complete.
+		 *
+		 * @since 5.3.4
```

Comment:
A build step will replace the `[version]` placeholder with the actual version number which we'll determine later. This may be included in 5.3.4 or maybe 5.4.0...

```suggestion
		 * @since [version]
```

Reply:
Done 👍